src/tsickle.ts

```diff
@@ -1500,7 +1502,27 @@ class Annotator extends ClosureRewriter {
     this.emit('\n');
     const name = node.name.getText();
 
-    this.emit(`/** @enum {number} */\n`);
+    // Figure out whether this is a string or number enum, by iterating through the enum
+    // members.  Note that TS allows mixed string/number enums also; just translate as {?}.
```

Comment:
On that code:
```
ERROR(4,3): : Enum member must have initializer.
```
so tsickle will never see it.

But this is legal:
```
enum X {
  B = 'x',
  A = 1,
  C,
}
```
and in that, C is 2.